packages/luna/src/components/Dialog/Dialog.tsx

```diff
@@ -1,67 +1,42 @@
-import React, { useCallback, useRef } from "react";
-import { Button, ButtonSize, ButtonType } from "@/components/Button";
-import { X } from "react-feather";
+import * as DialogComponent from "@radix-ui/react-dialog";
 import cn from "classnames";
+import React from "react";
+import { X } from "react-feather";
 
 export const Dialog: React.FC<{
+  trigger: React.ReactNode;
```

Comment:
I think the `trigger` should be optional because it will not be always the case that we will open the dialog via UI component (e.g., button).